src/meta/src/rpc/service/hummock_service.rs

```diff
@@ -204,6 +204,18 @@ where
         Ok(Response::new(RwReceiverStream::new(rx)))
     }
 
+    async fn report_compaction_task_progress(
```

Comment:
Suggest streaming interface

Reply:
I've proposed to implement it in next PR as this PR is quite long and its not so trivial to do the streaming as mentioned in this thread: https://github.com/singularity-data/risingwave/pull/4496#discussion_r944177447

Reply:
Agree with `streaming RPC`, shall we add some `TODO` comments for this so that we can easily locate it later.